src/command_line_parser.cc

```diff
@@ -525,26 +526,44 @@ std::vector<Option> TritonParser::recognized_options_
 #endif  // TRITON_ENABLE_METRICS
 #ifdef TRITON_ENABLE_TRACING
       {OPTION_TRACE_FILEPATH, "trace-file", Option::ArgStr,
-       "Set the file where trace output will be saved. If --trace-log-frequency"
+       "DEPRECATED: Please use --trace-config=triton,file=<path/to/your/file>"
+       " Set the file where trace output will be saved. If "
+       "--trace-log-frequency"
        " is also specified, this argument value will be the prefix of the files"
        " to save the trace output. See --trace-log-frequency for detail."},
       {OPTION_TRACE_LEVEL, "trace-level", Option::ArgStr,
+       "DEPRECATED: Please use --trace-config=level=<OFF|TIMESTAMPS|TENSORS>"
        "Specify a trace level. OFF to disable tracing, TIMESTAMPS to "
        "trace timestamps, TENSORS to trace tensors. It may be specified "
        "multiple times to trace multiple informations. Default is OFF."},
       {OPTION_TRACE_RATE, "trace-rate", Option::ArgInt,
+       "DEPRECATED: Please use --trace-config=rate=<rate value>"
        "Set the trace sampling rate. Default is 1000."},
       {OPTION_TRACE_COUNT, "trace-count", Option::ArgInt,
+       "DEPRECATED: Please use --trace-config=count=<count value>"
        "Set the number of traces to be sampled. If the value is -1, the number "
        "of traces to be sampled will not be limited. Default is -1."},
       {OPTION_TRACE_LOG_FREQUENCY, "trace-log-frequency", Option::ArgInt,
+       "DEPRECATED: Please use --trace-config=triton,log-frequency=<value>"
        "Set the trace log frequency. If the value is 0, Triton will only log "
        "the trace output to <trace-file> when shutting down. Otherwise, Triton "
        "will log the trace output to <trace-file>.<idx> when it collects the "
        "specified number of traces. For example, if the log frequency is 100, "
        "when Triton collects the 100-th trace, it logs the traces to file "
        "<trace-file>.0, and when it collects the 200-th trace, it logs the "
        "101-th to the 200-th traces to file <trace-file>.1. Default is 0."},
+      {OPTION_TRACE_CONFIG, "trace-config", "<string>,<string>=<string>",
+       "Specify global or trace mode specific configuration setting. "
+       "The format of this flag is --trace-config="
+       "<mode>,<setting>=<value>. "
+       "Where <mode> is either \"triton\" or \"opentelemetry\". "
+       "The default is \"triton\". To specify global trace settings "
+       "(level, rate, count, or mode), the format would be "
+       "--trace-config=<setting>=<value>. For \"triton\" mode, the server will "
```

Comment:
remove extra equals from here as well.

Reply:
Removed equal signs in `command_line_parser.cc` after all `--trace-config`